cmd/ping.go

```diff
@@ -19,14 +19,14 @@ Examples:
   # Ping google.com from 2 probes in New York
```

Comment:
Lets update the Long description too. Its weird I think and doesnt fit

Reply:
@jimaek I'm not sure I understand. Can you please provide a more adequate description ?

Reply:
I meant the one above this line, it didn't allow me to comment on the correct line.
`Long: The ping command sends an ICMP ECHO_REQUEST to obtain an ICMP ECHO_RESPONSE from a host or gateway.`

I dont think it follows our standard format and its not very helpful

Reply:
Ok I just updated to `The ping command allows sending ping requests to a target.`. If this is not what you're looking for please provide the exact copy

Reply:
Let's add "Often used to test the network latency and stability"

Reply:
Thanks, just added